docs/interfaces.md

```diff
@@ -518,3 +518,9 @@ to storage on the device
 Can query UPower for power devices, history and statistics.
 
 * Auto-Connect: yes
+
+### usb-raw
+
+Can directly send and receive data to a connected USB device.
```

Comment:
This doesn't limit to a particular device, so can we reword to be `Can directly send and receive data to connected USB devices.`

Sorry I didn't think of this before.